.drone.yml

```diff
@@ -11,18 +12,10 @@ workspace:
   path: gcp-vault
 
 steps:
-- name: mod-download
-  image: golang:1
-  commands:
-  - go mod download
-  environment:
-    GOPROXY: https://proxy.golang.org
-  depends_on:
-  - clone
-
-- name: test
-  image: golang:1
-  commands:
-  - go test -race -vet all -mod readonly ./...
-  depends_on:
-  - mod-download
+  - name: test
+    image: golang:1.19
+    commands:
+      - go test -v ./... -covermode=count .
```

Comment:
Just curious why you removed the `-race` flag?

Reply:
I ran the tests without the flag a few times locally, and it works fine without this flag. Perhaps it was an issue earlier, but is no longer an issue with newer versions of Go.

Reply:
`-race` should not resolve an issue, it enables the [race detector](https://go.dev/doc/articles/race_detector) in tests.

Reply:
gotcha, will retain it then.